src/parser.cpp

```diff
@@ -605,6 +605,81 @@ options_metadata parser::load_settings()
         "server.blacklist",
         value<config::authority::list>(&configured.server.blacklists),
         "Blocked client IP address, multiple entries allowed."
+    )
+    (
+        "web.web_sockets_ssl_certificate",
```

Comment:
[web] operates as a namespace for the settings, allowing the removal of "web_sockets" and "websocket" from their names. The names should therefore correspond to their [server] equivalents.

Reply:
Right, missed that completely.  Rename settings to server_settings and create additional web_settings?

Reply:
See the implementation of [forks] in blockchain.